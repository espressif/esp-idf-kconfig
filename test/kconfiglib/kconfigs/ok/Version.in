mainmenu "Test if version is parsed correctly"

    config VER
        string "Version"
        default 5.3

    config FOO
        int "Foo"
        default 1 if VER = 5.3
        default 0
