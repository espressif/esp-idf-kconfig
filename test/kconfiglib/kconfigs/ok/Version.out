#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#
CONFIG_VER="5.3"
CONFIG_FOO=1
